backend/db/queries/userQueries.js

```diff
@@ -1,13 +1,13 @@
-const db = require("../connection");
-const authHelpers = require("../../auth/helpers");
+const db = require('../connection');
```

Comment:
this is the problem with auto prettier type things. if one of you has it set to `"` and the other one set to `'` then it makes pull requests and commits over inflated.

Reply:
I see.. I guess we all have to have our settings to be the same to fix this

Reply:
yeah, put your settings in a file that is also committed to the repo, that way it is shared for the repo across your editing environments. for example for eslint you would have a `.eslintrc.json` file.

https://eslint.org/docs/user-guide/configuring